Index: SDL_mixer-1.2.9/music_cmd.c
===================================================================
--- SDL_mixer-1.2.9/music_cmd.c	(revision 9085)
+++ SDL_mixer-1.2.9/music_cmd.c	(working copy)
@@ -143,7 +143,11 @@
 /* Start playback of a given music stream */
 void MusicCMD_Start(MusicCMD *music)
 {
+#ifdef __uClinux__
+	music->pid = vfork();
+#else
 	music->pid = fork();
+#endif
 	switch(music->pid) {
 	    /* Failed fork() system call */
 	    case -1:
