diff -urp xenomai-2.4.93/configure xenomai-2.4.93.new/configure
--- xenomai-2.4.93/configure	2009-10-28 02:08:25.000000000 +0800
+++ xenomai-2.4.93.new/configure	2009-10-30 11:18:36.000000000 +0800
@@ -2463,9 +2463,9 @@ else
 	XENO_EMPTY_CFLAGS=false
 fi
 
-if eval test $includedir = /usr/include; then
-  as_fn_error "Using /usr/include as includedir is not supported. Please change your --prefix or specify another --includedir" "$LINENO" 5
-fi
+#if eval test $includedir = /usr/include; then
+#  as_fn_error "Using /usr/include as includedir is not supported. Please change your --prefix or specify another --includedir" "$LINENO" 5
+#fi
 
 ac_config_headers="$ac_config_headers src/include/xeno_config.h"
 
diff -urp xenomai-2.4.93/Makefile.in xenomai-2.4.93.new/Makefile.in
--- xenomai-2.4.93/Makefile.in	2009-10-28 02:08:25.000000000 +0800
+++ xenomai-2.4.93.new/Makefile.in	2009-10-30 12:54:33.000000000 +0800
@@ -795,7 +795,7 @@ echo '*** privileges.' ; \
           $$sudo mknod -m 666 $(DESTDIR)/dev/rtheap c 10 254 ; \
         fi
 
-install-exec-local: devices
+install-exec-local: #devices
 
 install-user:
 	$(MAKE) SUDO=false install
Only in xenomai-2.4.93.new: Makefile.in.orig
diff -urp xenomai-2.4.93/src/testsuite/clocktest/Makefile.am xenomai-2.4.93.new/src/testsuite/clocktest/Makefile.am
--- xenomai-2.4.93/src/testsuite/clocktest/Makefile.am	2009-09-21 01:05:40.000000000 +0800
+++ xenomai-2.4.93.new/src/testsuite/clocktest/Makefile.am	2009-10-30 11:18:36.000000000 +0800
@@ -11,7 +11,7 @@ clocktest_CPPFLAGS = -I$(top_srcdir)/inc
 clocktest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 
 clocktest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(testdir)
Only in xenomai-2.4.93.new/src/testsuite/clocktest: Makefile.am.orig
diff -urp xenomai-2.4.93/src/testsuite/clocktest/Makefile.in xenomai-2.4.93.new/src/testsuite/clocktest/Makefile.in
--- xenomai-2.4.93/src/testsuite/clocktest/Makefile.in	2009-10-28 02:08:28.000000000 +0800
+++ xenomai-2.4.93.new/src/testsuite/clocktest/Makefile.in	2009-10-30 11:18:36.000000000 +0800
@@ -243,7 +243,7 @@ clocktest_SOURCES = clocktest.c
 clocktest_CPPFLAGS = -I$(top_srcdir)/include/posix $(XENO_USER_CFLAGS) -I$(top_srcdir)/include
 clocktest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 clocktest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 EXTRA_DIST = runinfo.in
 all: all-am
Only in xenomai-2.4.93.new/src/testsuite/clocktest: Makefile.in.orig
diff -urp xenomai-2.4.93/src/testsuite/cyclic/Makefile.am xenomai-2.4.93.new/src/testsuite/cyclic/Makefile.am
--- xenomai-2.4.93/src/testsuite/cyclic/Makefile.am	2009-09-21 01:05:58.000000000 +0800
+++ xenomai-2.4.93.new/src/testsuite/cyclic/Makefile.am	2009-10-30 11:18:36.000000000 +0800
@@ -11,7 +11,7 @@ cyclictest_CPPFLAGS = -I$(top_srcdir)/in
 cyclictest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 
 cyclictest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(testdir)
Only in xenomai-2.4.93.new/src/testsuite/cyclic: Makefile.am.orig
diff -urp xenomai-2.4.93/src/testsuite/cyclic/Makefile.in xenomai-2.4.93.new/src/testsuite/cyclic/Makefile.in
--- xenomai-2.4.93/src/testsuite/cyclic/Makefile.in	2009-10-28 02:08:28.000000000 +0800
+++ xenomai-2.4.93.new/src/testsuite/cyclic/Makefile.in	2009-10-30 11:18:36.000000000 +0800
@@ -243,7 +243,7 @@ cyclictest_SOURCES = cyclictest.c
 cyclictest_CPPFLAGS = -I$(top_srcdir)/include/posix $(XENO_USER_CFLAGS) -DIPIPE_TRACE=1 -I$(top_srcdir)/include
 cyclictest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 cyclictest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 EXTRA_DIST = runinfo.in
 all: all-am
Only in xenomai-2.4.93.new/src/testsuite/cyclic: Makefile.in.orig
diff -urp xenomai-2.4.93/src/testsuite/irqbench/Makefile.am xenomai-2.4.93.new/src/testsuite/irqbench/Makefile.am
--- xenomai-2.4.93/src/testsuite/irqbench/Makefile.am	2009-09-21 01:06:10.000000000 +0800
+++ xenomai-2.4.93.new/src/testsuite/irqbench/Makefile.am	2009-10-30 11:18:36.000000000 +0800
@@ -21,7 +21,7 @@ irqloop_LDFLAGS = \
 	$(XENO_USER_LDFLAGS)
 
 irqloop_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 
 irqbench_SOURCES = irqbench.c
Only in xenomai-2.4.93.new/src/testsuite/irqbench: Makefile.am.orig
diff -urp xenomai-2.4.93/src/testsuite/irqbench/Makefile.in xenomai-2.4.93.new/src/testsuite/irqbench/Makefile.in
--- xenomai-2.4.93/src/testsuite/irqbench/Makefile.in	2009-10-28 02:08:28.000000000 +0800
+++ xenomai-2.4.93.new/src/testsuite/irqbench/Makefile.in	2009-10-30 11:18:36.000000000 +0800
@@ -258,7 +258,7 @@ irqloop_LDFLAGS = \
 	$(XENO_USER_LDFLAGS)
 
 irqloop_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 irqbench_SOURCES = irqbench.c
 irqbench_CPPFLAGS = \
Only in xenomai-2.4.93.new/src/testsuite/irqbench: Makefile.in.orig
diff -urp xenomai-2.4.93/src/testsuite/switchtest/Makefile.am xenomai-2.4.93.new/src/testsuite/switchtest/Makefile.am
--- xenomai-2.4.93/src/testsuite/switchtest/Makefile.am	2009-09-21 01:06:19.000000000 +0800
+++ xenomai-2.4.93.new/src/testsuite/switchtest/Makefile.am	2009-10-30 11:18:36.000000000 +0800
@@ -11,7 +11,7 @@ switchtest_CPPFLAGS = -I$(top_srcdir)/in
 switchtest_LDFLAGS =  $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 
 switchtest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(testdir)
Only in xenomai-2.4.93.new/src/testsuite/switchtest: Makefile.am.orig
diff -urp xenomai-2.4.93/src/testsuite/switchtest/Makefile.in xenomai-2.4.93.new/src/testsuite/switchtest/Makefile.in
--- xenomai-2.4.93/src/testsuite/switchtest/Makefile.in	2009-10-28 02:08:28.000000000 +0800
+++ xenomai-2.4.93.new/src/testsuite/switchtest/Makefile.in	2009-10-30 11:18:36.000000000 +0800
@@ -243,7 +243,7 @@ switchtest_SOURCES = switchtest.c
 switchtest_CPPFLAGS = -I$(top_srcdir)/include/posix $(XENO_USER_CFLAGS) -g -I$(top_srcdir)/include
 switchtest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 switchtest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 EXTRA_DIST = runinfo.in
 all: all-am
Only in xenomai-2.4.93.new/src/testsuite/switchtest: Makefile.in.orig
