diff -Nurp --exclude=.svn xenomai-2.4.x-clean/Makefile.in xenomai-2.4.x/Makefile.in
--- xenomai-2.4.x-clean/Makefile.in	2008-06-02 05:32:28.000000000 +0800
+++ xenomai-2.4.x/Makefile.in	2009-02-23 17:12:14.000000000 +0800
@@ -671,7 +671,7 @@ dist-hook:
 	test -e $(srcdir)/doc || rm -rf $(distdir)/doc
 
 dev devices:
-	@$(mkinstalldirs) $(DESTDIR)/dev; \
+	$(mkinstalldirs) $(DESTDIR)/dev; \
 	if test x`id -u` = x0; then \
            sudo= ; \
         else if test \! "x`type $(SUDO)`" = x ; then \
@@ -707,7 +707,7 @@ echo '*** privileges.' ; \
           $$sudo mknod -m 666 $(DESTDIR)/dev/rtheap c 10 254 ; \
         fi
 
-install-exec-local: devices
+install-exec-local: #devices
 
 install-user:
 	$(MAKE) SUDO=false install
diff -Nurp --exclude=.svn xenomai-2.4.x-clean/src/testsuite/clocktest/Makefile.am xenomai-2.4.x/src/testsuite/clocktest/Makefile.am
--- xenomai-2.4.x-clean/src/testsuite/clocktest/Makefile.am	2008-03-01 04:41:56.000000000 +0800
+++ xenomai-2.4.x/src/testsuite/clocktest/Makefile.am	2009-02-23 17:12:14.000000000 +0800
@@ -9,7 +9,7 @@ clocktest_CPPFLAGS = -I$(top_srcdir)/inc
 clocktest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 
 clocktest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(testdir)
diff -Nurp --exclude=.svn xenomai-2.4.x-clean/src/testsuite/clocktest/Makefile.in xenomai-2.4.x/src/testsuite/clocktest/Makefile.in
--- xenomai-2.4.x-clean/src/testsuite/clocktest/Makefile.in	2008-06-02 05:32:28.000000000 +0800
+++ xenomai-2.4.x/src/testsuite/clocktest/Makefile.in	2009-02-23 17:12:14.000000000 +0800
@@ -232,7 +232,7 @@ clocktest_SOURCES = clocktest.c
 clocktest_CPPFLAGS = -I$(top_srcdir)/include/posix $(XENO_USER_CFLAGS) -I$(top_srcdir)/include
 clocktest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 clocktest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 EXTRA_DIST = runinfo.in
 all: all-am
diff -Nurp --exclude=.svn xenomai-2.4.x-clean/src/testsuite/cyclic/Makefile.am xenomai-2.4.x/src/testsuite/cyclic/Makefile.am
--- xenomai-2.4.x-clean/src/testsuite/cyclic/Makefile.am	2008-03-01 04:41:56.000000000 +0800
+++ xenomai-2.4.x/src/testsuite/cyclic/Makefile.am	2009-02-23 17:12:14.000000000 +0800
@@ -9,7 +9,7 @@ cyclictest_CPPFLAGS = -I$(top_srcdir)/in
 cyclictest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 
 cyclictest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(testdir)
diff -Nurp --exclude=.svn xenomai-2.4.x-clean/src/testsuite/cyclic/Makefile.in xenomai-2.4.x/src/testsuite/cyclic/Makefile.in
--- xenomai-2.4.x-clean/src/testsuite/cyclic/Makefile.in	2008-06-02 05:32:28.000000000 +0800
+++ xenomai-2.4.x/src/testsuite/cyclic/Makefile.in	2009-02-23 17:12:14.000000000 +0800
@@ -232,7 +232,7 @@ cyclictest_SOURCES = cyclictest.c
 cyclictest_CPPFLAGS = -I$(top_srcdir)/include/posix $(XENO_USER_CFLAGS) -DIPIPE_TRACE=1 -I$(top_srcdir)/include
 cyclictest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 cyclictest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 EXTRA_DIST = runinfo.in
 all: all-am
diff -Nurp --exclude=.svn xenomai-2.4.x-clean/src/testsuite/irqbench/Makefile.am xenomai-2.4.x/src/testsuite/irqbench/Makefile.am
--- xenomai-2.4.x-clean/src/testsuite/irqbench/Makefile.am	2008-03-01 04:41:56.000000000 +0800
+++ xenomai-2.4.x/src/testsuite/irqbench/Makefile.am	2009-02-23 17:12:14.000000000 +0800
@@ -19,7 +19,7 @@ irqloop_LDFLAGS = \
 	$(XENO_USER_LDFLAGS)
 
 irqloop_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 
 irqbench_SOURCES = irqbench.c
diff -Nurp --exclude=.svn xenomai-2.4.x-clean/src/testsuite/irqbench/Makefile.in xenomai-2.4.x/src/testsuite/irqbench/Makefile.in
--- xenomai-2.4.x-clean/src/testsuite/irqbench/Makefile.in	2008-06-02 05:32:28.000000000 +0800
+++ xenomai-2.4.x/src/testsuite/irqbench/Makefile.in	2009-02-23 17:12:14.000000000 +0800
@@ -247,7 +247,7 @@ irqloop_LDFLAGS = \
 	$(XENO_USER_LDFLAGS)
 
 irqloop_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 irqbench_SOURCES = irqbench.c
 irqbench_CPPFLAGS = \
diff -Nurp --exclude=.svn xenomai-2.4.x-clean/src/testsuite/switchtest/Makefile.am xenomai-2.4.x/src/testsuite/switchtest/Makefile.am
--- xenomai-2.4.x-clean/src/testsuite/switchtest/Makefile.am	2008-03-01 04:41:56.000000000 +0800
+++ xenomai-2.4.x/src/testsuite/switchtest/Makefile.am	2009-02-23 17:12:14.000000000 +0800
@@ -9,7 +9,7 @@ switchtest_CPPFLAGS = -I$(top_srcdir)/in
 switchtest_LDFLAGS =  $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 
 switchtest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(testdir)
diff -Nurp --exclude=.svn xenomai-2.4.x-clean/src/testsuite/switchtest/Makefile.in xenomai-2.4.x/src/testsuite/switchtest/Makefile.in
--- xenomai-2.4.x-clean/src/testsuite/switchtest/Makefile.in	2008-06-02 05:32:28.000000000 +0800
+++ xenomai-2.4.x/src/testsuite/switchtest/Makefile.in	2009-02-23 17:12:14.000000000 +0800
@@ -232,7 +232,7 @@ switchtest_SOURCES = switchtest.c
 switchtest_CPPFLAGS = -I$(top_srcdir)/include/posix $(XENO_USER_CFLAGS) -g -I$(top_srcdir)/include
 switchtest_LDFLAGS = $(XENO_POSIX_WRAPPERS) $(XENO_USER_LDFLAGS)
 switchtest_LDADD = \
-	../../skins/posix/libpthread_rt.la -lpthread -lrt
+	-lpthread -lrt ../../skins/posix/libpthread_rt.la
 
 EXTRA_DIST = runinfo.in
 all: all-am
