Index: btsco-0.5/alsa-plugins/headsetd/headsetd.c
===================================================================
--- btsco-0.5/alsa-plugins/headsetd/headsetd.c	(revision 9358)
+++ btsco-0.5/alsa-plugins/headsetd/headsetd.c	(working copy)
@@ -55,10 +55,17 @@
 	}
 
 	if (daemonize) {
+#ifdef __uClinux__
+		int ret = vfork();
+		if (ret > 0) {
+			_exit(0);
+		}
+#else
 		int ret = fork();
 		if (ret > 0) {
 			exit(0);
 		}
+#endif
 		else if(ret < 0) {
 			perror("Unable to fork");
 			exit(1);
